core/store/src/flat_state.rs

```diff
@@ -177,18 +193,16 @@ mod imp {
             }
         }
 
+        // TODO (#7327): change the function signature to Result<FlatStorageState, Error> when
+        // we stabilize feature protocol_feature_flat_stote. We use option now to return None when
```

Comment:
```suggestion
        // we stabilize feature protocol_feature_flat_state. We use option now to return None when
```
just to make sure grepping works...